AcceptanceTests/attach_screenshots/attach-screenshots.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+### Run this script to attach the screenshots
+### Can pass in any parameters, such as --env "RELEASEID=$(Release.ReleaseId)"
+
+#Set the path to the root of the test project folder from where you are running this file from
+PATH_TO_ROOT=.
+
+if [ ! -z "$1" ]
+  then
+    echo "### You provided the arguments:" "$@"
+fi
+
+echo "### Running from " $(pwd)
+
+echo "### Building docker image"
+docker build -t screenshots -f attach_screenshots/AttachScreenshots.dockerfile .
+
+echo "### Running docker image"
+docker run $@ screenshots
```

Comment:
Add `--rm`  flag to delete the container after running
E.g. `docker run --rm $@ screenshots`